tests/explainers/test_linear.py

```diff
@@ -35,6 +35,7 @@ def test_tied_pair_new():
     assert np.abs(explainer.shap_values(X) - np.array([0.5, 0.5, 0])).max() < 0.05
 
 def test_wrong_masker():
+    pytest.importorskip("cv2")
     with pytest.raises(NotImplementedError):
         shap.explainers.Linear((0, 0), shap.maskers.Image("blur(10,10)", (10, 10, 3)))
```

Comment:
The point of this test is to check that an error is raised if an invalid masker is passed in, not really about the Image masker.

So instead of skipping because `cv2` doesn't exist, let's just change the masker to something other than the accepted ones (which I think is Partition, Exact, and Independent?) which doesn't depend on any extra dependencies, if possible?

Reply:
I can take a quick look at this. I can raise a new issue and address this in a new PR.

Reply:
See #64.

Reply:
Fixed by #67